### DISCUSSION TITLE
hii i wanat to download stake predictor if there is one

### DISCUSSION THREAD

**Discussion by REGALREPAIRSHOP** [community/community#122368] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hii i wanat to get to know about stake predictor
can you contact lcvpike@gmail.com

**Comment by ebndev** [comment]
Hi @REGALREPAIRSHOP,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from [stake predictor](https://github.com/topics/stake-hack-predictor). You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
